edx_proctoring/api.py

```diff
@@ -792,6 +794,8 @@ def update_attempt_status(exam_id, user_id, to_status, raise_if_not_found=True,
     )
     if add_start_time:
         exam_attempt_obj.started_at = datetime.now(pytz.UTC)
+    elif alias_timeout:
```

Comment:
Nit: while you're in here, could you rename this variable as I found it very hard to understand what it means. In what way is it an alias?